Tabloid/client/src/components/ApplicationViews.js

```diff
@@ -19,7 +20,10 @@ export default function ApplicationViews({ isLoggedIn }) {
           <CategoryList />
         </Route>
         <Route path="/posts" exact>
-          {isLoggedIn ? <PostList /> : <Redirect to="/login" />}
+          <PostList />
+        </Route>
+        <Route path="/posts/:postId" exact>
```

Comment:
1 - Does this route path need to exact?
2 - Should this route param be an integer? if so, you can add some regex to make sure it is an integer.. see client side chapters on dynamic routing for examples!